docs/concepts/capabilities.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Capabilities'
+icon: 'ti-anchor' # themify icon pack : https://themify.me/themify-icons
+description: 'Application declaration, deployment, and management'
+sidebar_position: 1
+type: 'docs'
+---
+
+## Overview
+
+A ***capability** is a pluggable piece of functionality for an application. More specifically, capabilities are common, abstracted functionalities&mdash;usually the sort of utilities that aren't core business logic for a given application, such as...
+
+* HTTP Client & Server
+* Logging
+* Key-Value Storage
+* Messaging
+* Blob Storage
+* Secrets
+
+You can find a complete list of first-party capabilities in the [Capability Catalog](/docs/capabilities/index.mdx).
+
+## How capabilities work
+
+Capabilities are enacted by an **interface** and one or more swappable **capability providers**. A provider delivers a specific implementation of a given capability by communicating with other entities in the common tongue of a standard API. 
```

Comment:
Link to WIT documentation (ours) here? Not sure if we want to mention WIT or if it's too much.

Reply:
Yeah, I think this is about the right point to raise it. (Though I think it's probably too much for the top-level catalog page.)